source/query-bar.txt

```diff
@@ -323,9 +306,23 @@ as favorites. To view the list of queries saved as favorites, click on
 .. figure:: /images/compass/query-history-favorite.png
    :figwidth: 316px
 
+Limitations
+-----------
+
+- The query bar options do not support the :manual:`$near
+  </reference/operator/query/near/>` and
+  :manual:`$nearSphere </reference/operator/query/nearSphere/>` 
+  geospatial query operators, because they do not provide any additional 
+  functionality in the :ref:`schema view <schema-tab>`.
+
+- The :ref:`Project <query-bar-project>` query bar option does not 
```

Comment:
Project does support $slice in the released 1.18.0-beta.0 - just to note

Reply:
Noted. Tracking with DOCSP-4709.